index.js

```diff
@@ -448,6 +449,43 @@ module.exports = function(options) {
     return self.callAllAndEmit('afterInit', 'afterInit', callback);
   }
 
+  function lintModules(callback) {
+    _.each(self.modules, function(module, name) {
+      if (name.match(/-widgets$/) && (!extending(module)) && (!module.options.fromScratch)) {
+        lint('The module ' + name + ' does not extend anything.\n\nA widgets module usually extends `apostrophe-widgets` or\n`apostrophe-pieces-widgets`. Or possibly you forgot to npm install something.\n\nIf you are sure you are doing the right thing, set the `fromScratch` option\nto `true` for this module.');
```

Comment:
For pages you use the `-pages` format, "a -pages module," but here you don't lead with the dash, "a widgets module." We should do one or the other. As with pages, I think quotes would be good around "-widgets" to indicate it's related to the name.